src/xcode/ENA/ENA/Source/Views/GradientView.swift

```diff
@@ -46,6 +51,7 @@ class GradientView: UIView {
 		case lightBlue(withStars: Bool)
 		case mediumBlue(withStars: Bool)
 		case darkBlue(withStars: Bool)
+		case certificateCheck
```

Comment:
Naming does not fit to the other types.

Reply:
Renamed it to take the context out and just reflect the colors as we did for the others